docs/openapi.yaml

```diff
@@ -0,0 +1,826 @@
+    documents to PDF.
+externalDocs:
+  url: https://gotenberg.dev/docs/about
+servers:
+  - url: 'http://localhost:3000'
+    description: Local server with the default Docker image and port
+tags:
+  - name: chromium
+    description: Operations of the Chromium module
+    externalDocs:
+      url: https://gotenberg.dev/docs/modules/chromium
+  - name: libreoffice
+    description: Operations of the Libreoffice module
+    externalDocs:
+      url: https://gotenberg.dev/docs/modules/libreoffice
+  - name: pdfengines
+    description: Operations of the PDF Engines module
+    externalDocs:
+      url: https://gotenberg.dev/docs/modules/pdf-engines
+paths:
+  /forms/chromium/convert/url:
+    post:
+      tags:
+        - chromium
+      summary: Convert the contents of a given URL to PDF
+      description: >-
+        Send a remote URL in your API request via the `remoteURL` parameter, and
+        get the resulting PDF file. The API will fetch the given URL and render
+        the page to PDF using the underlying headless Chrome instance.
+      parameters:
+        - in: header
+          name: Gotenberg-Output-Filename
+          description: >-
+            By default, the API generates a UUID filename.
+            However, you may also specify the filename per request,
+            thanks to the Gotenberg-Output-Filename header.
+            Caution! The API adds the file extension automatically; you don't have to set it.
+          schema:
+            type: string
+          required: false
+        - in: header
+          name: Gotenberg-Trace
+          description: >-
+            The trace, or request ID, identifies a request in the logs.
+
+            By default, the API generates a UUID trace for each request.
+            However, you may also specify the trace per request, thanks to the Gotenberg-Trace header.
+          schema:
+            type: string
+          required: false
+      requestBody:
+        required: true
+        description: >-
+          The URL conversion request has to be a `multipart/form-data` request
+          that includes a `remoteURL`. The API uses a headless Chrome instance
+          to do the conversion, therefore print parameter such as margins and
+          paper size are also accepted as optional parameters.
+        content:
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/URLConvertRequestBody'
+            examples: { }
+      responses:
+        '200':
+          $ref: '#/components/responses/SuccessfulPDF'
+        '400':
+          description: Bad Request
+
+  /forms/chromium/convert/html:
+    post:
+      tags:
+        - chromium
+      summary: Convert a given HTML file to PDF
+      description: >-
+        Send an HTML file called `index.html` as a multipart form request, and
+        get the resulting PDF file. You can optionally include `header.html` and
+        `footer.html` files as part of the request as well.
```

Comment:
One may include the header/footer in all Chromium endpoints ^^

Reply:
added this information to all the chromium endpoints